java/test/apps/gui3/dp3/DecoderPro3Test.java

```diff
@@ -13,45 +13,50 @@
  *
  * Description: Tests for the DecoderPro3 application.
  *
- * @author  Paul Bender Copyright (C) 2016
+ * @author Paul Bender Copyright (C) 2016
  */
 public class DecoderPro3Test {
 
     @Test
     public void testCtor() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());        
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         String[] args = {"DecoderProConfig3.xml"};
-        apps.AppsBase a = new DecoderPro3(args){
+        AppsBase a = new DecoderPro3(args) {
             // force the application to not actually start.  
             // Just checking construction.
-            @Override 
-            protected void start(){}
-           @Override
-            protected void configureProfile(){
-                 JUnitUtil.resetInstanceManager();
+            @Override
+            protected void start() {
+            }
+
+            @Override
+            protected void configureProfile() {
+                JUnitUtil.resetInstanceManager();
```

Comment:
Only if a known profile is needed for the test. This method that's being overridden exists so that other classes that need a profile can get it, but if the class being tested does not need it, I don't think there is any need to set it.